mypy/test/data.py

```diff
@@ -561,13 +549,37 @@ def pytest_addoption(parser: Any) -> None:
 def pytest_pycollect_makeitem(collector: Any, name: str, obj: Any) -> Any:
     if not isinstance(obj, type) or not issubclass(obj, DataSuite):
         return None
+    if obj is DataSuite:
```

Comment:
Add comment here -- something like this perhaps: only classes derived from `DataSuite` contain test cases, not the `DataSuite` class.